COBOL Programming Course #1 - Getting Started/COBOL Programming Course #1 - Getting Started.md

```diff
@@ -177,42 +177,48 @@ Follow these steps:
 
 1. Navigate to the Zowe Explorer tree on the right side and look for the + sign.
 
-![](Images/code4z-img8.jpg)
+![](Images/code4z/code4z-img8.png)
 
 2. Click on the + sign.
 A dialog box will appear and ask if you want to "Create a New Connection to z/OS".
 
 3. Press enter or click on that selection.
 
-![](Images/code4z-img9.png)
+![](Images/code4z/code4z-img9.png)
 
 4. Enter a Profile name in the "Connection Name" field.
 
-![](Images/code4z-img10.png)
+![](Images/code4z/code4z-img10.png)
 
 5. Enter the URL and Port that you received by email when you registered for the COBOL Course. The connection information that you need has a title of "IP address for VSCode extension".
 
-![](Images/code4z-img11.png)
+![](Images/code4z/code4z-img11.png)
 
 6. Enter your Username. This is also included in the email.
 
-![](Images/code4z-img12-18.png)
+![](Images/code4z/code4z-img12-18.png)
 
 Note: You can leave this blank if you do not want to save your credentials in your machine. You will be prompted for your username once you start using Zowe Explorer.
 
 7. Enter your Password.
 
-![](Images/code4z-img13-19.png)
+![](Images/code4z/code4z-img13-19.png)
 
 Note: You can leave this blank if you do not want to save your credentials in your machine. You will be prompted for your username once you start using Zowe Explorer.
 
 8. Select True/False if you want to accept or reject Self-Signed Certificates. For this course, please select false.
 
-![](Images/code4z-img14-20.png)
+![](Images/code4z/code4z-img14-20.png)
+
+![](Images/image073-1.png)
+
+![](Images/image073-2.png)
+
+![](Images/image073-3.png)
```

Comment:
So, this steps :)

Reply:
Wait, is there any step missing? If I recall correctly, there used to be a base path step here... hmmmm

Reply:
yes, you are right.
I will add it, and the accompanying steps for the images